src/App.tsx

```diff
@@ -15,37 +18,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortMethod {
+  Alphabetically,
```

Comment:
```suggestion
  Alphabetically = 'Alphabetically',
```

Would be good to add value everywhere